### PR TITLE
Migrate all code and data from Tokyo* to postgres

### PR DESCRIPTION
**Pull request by clint-tseng**
Please see #87 for the relevant ticket, and the commit messages herein for further detail.

### CODE REVIEW

File: model/connection_manager.rb [hunk 1]
```diff
@@ -19,28 +19,24 @@ def call(env)
...
-  end
-
   def add_finalizer_hook!
     at_exit do
       begin
-        self.class.connection.each_value{ |connection| connection.close }
+        self.class.db.disconnect
       rescue Exception => e
-        puts "Error closing Tokyo Cabinet connection. You might have to clean up manually."
+        puts "Error closing Postgres connection. Some minor data loss may have occurred for in-flight transactions."
```
Comment (lognaturel) [inline model/connection_manager.rb]: This is not very friendly to non-technical users who will think about ✈️. Why not just "ongoing transfers" or something like that?

File: model/user.rb [hunk 2]
```diff
@@ -10,50 +10,61 @@ class User
...
 
-    return result
+    return nil if data.nil?
+    return (User.new key, data)
   end
 
+# Class
   def self.create(data)
     key = data[:username].downcase
     pepper = Time.now.to_f.to_s
```
Comment (lognaturel) [inline model/user.rb]: Cute.

File: model/connection_manager.rb (hunk 1 above)
Comment (clint-tseng) [inline model/connection_manager.rb]: this is a STDERR logging message, not a user-facing one.

**Comment by clint-tseng** [comment]
@lognaturel account creation is now considered a login with c3de5d2.

**Comment by clint-tseng** [comment]
And no—form UIDs are fully unique and meaningful in their own domain.

File: model/connection_manager.rb (hunk 1 above)
Comment (lognaturel) [inline model/connection_manager.rb]: Oh, right. Carry on, then.

**Comment by lognaturel** [comment]
The form UID feels really short but I suppose 56 billion possibilities is enough. I haven't checked but I presume you check for uniqueness on creation.

Want me to merge, or...?

**Comment by clint-tseng** [comment]
Aye, it is ensured both with a db limit as well as here: https://github.com/opendatakit/build/pull/88/files#diff-12a1474e5ac1eecd24f810acb3e9613fR40

I'll merge it tomorrow.

**Comment by lognaturel** [comment]
Awesome. Evidently my skimming was not that thorough.

**Pull request merged**